app/Services/UserInfoService.php

```diff
@@ -0,0 +1,50 @@
+<?php
+namespace App\Services;
+
+use App\ModelRepositoryInterfaces\UserInfoModelRepositoryInterface;
+use App\Models\UserInfo;
+use App\Contracts\UserInfoContract;
+use Validator;
+
+
+class UserInfoService implements UserInfoContract
+{
+    protected $userInfoModelRepo;
+
+    public function __construct(UserInfoModelRepositoryInterface $userInfoModelRepo){
+        $this->userInfoModelRepo = $userInfoModelRepo;
+    }
+
+    public function getStudentsByGradDate($graddate)
+    {
+        return $this->userInfoModelRepo->getStudentsByGradDate($graddate);
+    }
+
+    public function getStudentsByCollege($collegename)
+    {
+        return $this->userInfoModelRepo->getStudentsByCollege($collegename);
+    }
+
+    public function getStudentsByMajor($majorname)
+    {
+        return $this->userInfoModelRepo->getStudentsByMajor($majorname);
+    }
+
+    public function store($data)
```

Comment:
I would revisit the naming of this method to something like: "createStudentInfo"